distribution/feature-packs/server-feature-pack/src/main/resources/licenses/keycloak/licenses.xml

```diff
@@ -103,7 +103,7 @@
     <dependency>
       <groupId>org.codehaus.plexus</groupId>
       <artifactId>plexus-utils</artifactId>
-      <version>3.0.22</version>
+      <version>3.1.1</version>
```

Comment:
Why this upgrade?

Reply:
@martin-kanis Same reason like for the `picketbox` version bump. It comes from Wildfly Core update IIRC (that's why it got updated too).

Reply:
Spoken with @martin-kanis further about this. And looks there's a licenses dependency cleanup necessary. It will be done in the future (tracked via [KEYCLOAK-14792](https://issues.redhat.com/browse/KEYCLOAK-14792) issue)